executers/utils/gopath_test.go

```diff
@@ -0,0 +1,58 @@
+package utils
+
+import (
+	"github.com/apex/log"
+	"runtime"
+	"strings"
+	"testing"
+)
+
+func TestParseGoPathWindows(t *testing.T) {
```

Comment:
This test and the *TestParseGoPathUnix* test share a lot of duplicated code which can be probably reused.

Reply:
Done